### ISSUE TITLE
Resizing window breaks background image

### ISSUE THREAD

**Issue by Kurtsley** [cxong/cdogs-sdl#859]
When the dynamic gameplay background is shown (before the campaign is selected), the screen can be resized with no issues. When a static image is shown, resizing the window changes the background to a solid color. The color changes based on the image. Once the window has been resized, the original image does not appear again.

Ex:
<img src="https://github.com/cxong/cdogs-sdl/assets/73447098/28f22494-bd4a-43aa-9d75-d49e8e75a11f" width=500>

**Comment by BROCODES2024** [comment]
Hi is this eligible for Hacktoberfest 2024? 
If yes i would like to work on it, please provide more details about the issue

**Comment by cxong** [comment]
Yes this is eligible, but it may be too complex for hacktoberfest. When the window is resized, `HasResolutionChanged` is set to true, but if you look for references to it, it is handled in the mainmenu (with live background), in-game, editor, but not in the other menus. There are many other menus, like player selection, weapon selection, mission briefing etc, and it is not clear if there is a central place where this fix can be made. A fix attempt would need to include a design review before a code fix.

**Comment by BROCODES2024** [comment]
That's really too complex for me as a beginner, do you have any other issues which are on the easier side?

**Comment by cxong** [comment]
check issues labelled "good first issue"
